Modix.Services/Core/AuthorizationService.cs

```diff
@@ -28,6 +28,11 @@ public interface IAuthorizationService
         /// </summary>
         ulong? CurrentGuildId { get; }
 
+        /// <summary>
+        /// Indicates whether the current user (if any) should be ignored during authorization checks.
+        /// </summary>
+        bool? CurrentUserIgnoresAuthorization { get; }
```

Comment:
I would just make this a regular `bool`. It'll always be false when there's no authenticated user, which shouldn't break any other logic - if there's no user, they can't ignore authorization.